README.md

```diff
@@ -13,6 +13,11 @@ Read more in [this blog post](https://medium.com/@amirziai/a-flask-api-for-servi
 pip install -r requirements.txt
 ```
 
+### Running API
+```
+python2 main.py 9999
```

Comment:
Can you please indicate that port is an argument. I think the convention is `[port]`. Not sure.